pipelines/optimus/Optimus.wdl

```diff
@@ -110,19 +112,30 @@ workflow Optimus {
         annotations_gtf = annotations_gtf
     }
 
-    call CorrectUmiMarkDuplicates.SortAndCorrectUmiMarkDuplicates {
+    call Picard.SortBamAndIndex as PreUMISort {
```

Comment:
So the UMI tools require sorting before, and the output doesn't preserve the sort order? It seems odd to have to call `Picard.SortBamAndIndex` both before and after mark duplicates.

Reply:
It normally prints out sorted output, but I've used a flag that turns it off. The rational here that its probably not the most efficient way to sort and more importantly that it reduces the run duration of the step. 

Reducing the run duration has two advantages: (1) It reduces chances of pre-emption in prod (2) it allows the two different types of sorting required downstream (sort by cell and sort by gene) to start immediately  [This sorting does nested sorting of tags and there is no reason to wait for the positional sorting to complete]. The positional sorting is only required to generate the bam.